lib/pmd.js

```diff
@@ -244,6 +244,7 @@ pmd.getArguments = function(process){
  */
 pmd.generateData = function(config){
   var objs = [];
+  var indexData = [];
```

Comment:
I don't understand your question, but the goal is to be able to include a TOC in all files for better navigation. Example of a custom HTML template I used with this pull request:

![image](https://cloud.githubusercontent.com/assets/5391291/20835819/820fec44-b86a-11e6-9d0a-4ce4215da083.png)